testcharms/charm-repo/quantal/lxd-profile-alt/lxd-profile.yaml

```diff
@@ -25,9 +25,10 @@ devices:
     type: usb
     vendorid: 0fce
     productid: 51da
-  bdisk:
-    type: unix-block
-    source: /dev/loop0
+# unix-block failing on nested containers
+#  bdisk:
+#    type: unix-block
+#    source: /dev/loop0
```

Comment:
Think we should just remove this, keeps the code simpler to grok.

Reply:
How should we handle profiles for different configs?  So far we have 3 different that I'm aware of: localhost, nested localhost and lxd on aws.  Will we miss anything by going with the lowest common denominator?

Reply:
per conversation, put back to original, for unit tests.  we need a different solution for manual testing, perhaps a small go program to write out different types of charm based on known permutations.